pkg/apis/externaldns/types_test.go

```diff
@@ -29,77 +29,79 @@ import (
 	}
 
 	overriddenConfig = &Config{
-		Master:               "http://127.0.0.1:8080",
-		KubeConfig:           "/some/path",
-		Sources:              []string{"service", "ingress"},
-		Namespace:            "namespace",
-		FQDNTemplate:         "{{.Name}}.service.example.com",
-		Compatibility:        "mate",
-		Provider:             "google",
-		GoogleProject:        "project",
-		DomainFilter:         []string{"example.org", "company.com"},
-		ZoneIDFilter:         []string{"/hostedzone/ZTST1", "/hostedzone/ZTST2"},
-		AWSZoneType:          "private",
-		AWSAssumeRole:        "some-other-role",
-		AzureConfigFile:      "azure.json",
-		AzureResourceGroup:   "arg",
-		CloudflareProxied:    true,
-		InfobloxGridHost:     "127.0.0.1",
-		InfobloxWapiPort:     8443,
-		InfobloxWapiUsername: "infoblox",
-		InfobloxWapiPassword: "infoblox",
-		InfobloxWapiVersion:  "2.6.1",
-		InfobloxSSLVerify:    false,
-		InMemoryZones:        []string{"example.org", "company.com"},
-		PDNSServer:           "http://ns.example.com:8081",
-		PDNSAPIKey:           "some-secret-key",
-		Policy:               "upsert-only",
-		Registry:             "noop",
-		TXTOwnerID:           "owner-1",
-		TXTPrefix:            "associated-txt-record",
-		Interval:             10 * time.Minute,
-		Once:                 true,
-		DryRun:               true,
-		LogFormat:            "json",
-		MetricsAddress:       "127.0.0.1:9099",
-		LogLevel:             logrus.DebugLevel.String(),
+		Master:                "http://127.0.0.1:8080",
+		KubeConfig:            "/some/path",
+		Sources:               []string{"service", "ingress", "connector"},
+		Namespace:             "namespace",
+		FQDNTemplate:          "{{.Name}}.service.example.com",
+		Compatibility:         "mate",
+		Provider:              "google",
+		GoogleProject:         "project",
+		DomainFilter:          []string{"example.org", "company.com"},
+		ZoneIDFilter:          []string{"/hostedzone/ZTST1", "/hostedzone/ZTST2"},
+		AWSZoneType:           "private",
+		AWSAssumeRole:         "some-other-role",
+		AzureConfigFile:       "azure.json",
+		AzureResourceGroup:    "arg",
+		CloudflareProxied:     true,
+		InfobloxGridHost:      "127.0.0.1",
+		InfobloxWapiPort:      8443,
+		InfobloxWapiUsername:  "infoblox",
+		InfobloxWapiPassword:  "infoblox",
+		InfobloxWapiVersion:   "2.6.1",
+		InfobloxSSLVerify:     false,
+		InMemoryZones:         []string{"example.org", "company.com"},
+		PDNSServer:            "http://ns.example.com:8081",
+		PDNSAPIKey:            "some-secret-key",
+		Policy:                "upsert-only",
+		Registry:              "noop",
+		TXTOwnerID:            "owner-1",
+		TXTPrefix:             "associated-txt-record",
+		Interval:              10 * time.Minute,
+		Once:                  true,
+		DryRun:                true,
+		LogFormat:             "json",
+		MetricsAddress:        "127.0.0.1:9099",
+		LogLevel:              logrus.DebugLevel.String(),
+		ConnectorSourceServer: "localhost:8080",
```

Comment:
The idea is to test whether the flags work. In other words, this should be different from what's in `DefaultConfig`.